### PR TITLE
unpin ragas

### LINKED ISSUE #674: Ragas: update pinned version
We are pinning an old version of Ragas (0.1.1).

We should try to use the latest version and see what we need to change to support this.

### PR DESCRIPTION
**Pull request by anakin87**
fixes #674

@shadeMe please take a look.
The tests are passing but you mentioned some changes were breaking our integration...

### CODE REVIEW

File: integrations/ragas/src/haystack_integrations/components/evaluators/ragas/metrics.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     ContextUtilization,  # type: ignore
     Faithfulness,  # type: ignore
 )
-from ragas.metrics.base import Metric  # type: ignore
+from ragas.metrics.base import Metric
```
Comment (anakin87) [inline integrations/ragas/src/haystack_integrations/components/evaluators/ragas/metrics.py:19]: unrelated

**Review by shadeMe — APPROVED** [review]
LGTM if the tests pass!
